ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -122,6 +129,27 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
+    /// A layout guide that covers the on-screen portion of the current commanding view.
+    @objc public let layoutGuide = UILayoutGuide()
```

Comment:
is this layoutguide only applicaable to bottomsheetviewcontroller?

Reply:
It will wrap both the bar and the sheet, depending on screen size.

Reply:
Can we be a bit more specific in the name? commandingLayoutGuide?